label_studio/ml/api.py

```diff
@@ -88,6 +88,15 @@ def perform_create(self, serializer):
         ml_backend = serializer.save()
         ml_backend.update_state()
 
+        project = ml_backend.project
+
+        # In case we are adding the model, let's set it as the default
+        # to obtain predictions. This approach is consistent with uploading
+        # offline predictions, which would be set automatically.
+        if project.show_collab_predictions and (project.model_version is None or project.model_version == ''):
```

Comment:
this latter conjunct is the same as `not project.model_version`

Reply:
yeah, I also thought so!

Reply:
```suggestion
        if project.show_collab_predictions and not project.model_version:
```